### ISSUE TITLE
In Modules, can't find latest release of better-sqlite3

### ISSUE THREAD

**Issue by linonetwo** [electron/fiddle#1444]
<img width="298" alt="截屏2023-08-22 00 21 41" src="https://github.com/electron/fiddle/assets/3746270/7cc7e825-4677-425c-9af2-f0b5328715be">

I'm reproducing a bug that better-sqlite3 crush electron after electron upgrade from 25.2.0 to 26.0.0

But I can't find the `"better-sqlite3": "^8.5.1"` I need in the Modules list, and there is no way to modify it.

**Comment by dsanders11** [comment]
Thanks for the report. This is unfortunately an upstream issue where the search index is at times not being updated (for extended periods of time). We should look into a UI change to balance providing results from the search index with the ability to manually override it in situations like this.

**Comment by erickzhao** [comment]
Yes, this is an upstream issue with https://replicate.npmjs.org, which powers our Algolia search index.

See:
https://github.com/npm/registry/blob/master/docs/REPLICATE-API.md
https://github.com/npm/feedback/discussions/937
